src/Features/CSharp/Portable/Completion/CompletionProviders/OperatorsAndIndexer/ConversionCompletionProvider.cs

```diff
@@ -119,28 +120,30 @@ private ImmutableArray<CompletionItem> GetBuiltInEnumConversions(SemanticModel s
             // No suggestion for the other two kinds of conversions:
             // * From sbyte, byte, short, ushort, int, uint, long, ulong, char, float, double, or decimal to any enum_type.
             // * From any enum_type to any other enum_type.
-            if (!container.IsEnumType())
+            if (container.IsEnumMember() || // SomeEnum.EnumValue.$$
+                (container.IsEnumType() &&  // someEnumVariable.$$
+                    !(semanticModel.GetSymbolInfo(expression, cancellationToken).Symbol is ITypeSymbol typeSymbol && typeSymbol.IsEnumType()))) // but not SomeEnum.$$
```

Comment:
condition is too complex.  it's esp. bad that it now requires the inline comments. please just use normal commented statements .